src/pages/itemsPage/ProductIdPage/components/productCommentSection/ProductCommentSection.jsx

```diff
@@ -0,0 +1,47 @@
+import { getProductCommentList } from "@/api/productFetch";
+import { PATH } from "@/app/router";
+import EmptyImg from "@/assets/imgs/img_inquiry_empty.png";
+import BackSvg from "@/assets/svg/BackSvg";
+import { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import CommentForm from "./CommentForm";
+import CommentItem from "./CommentItem";
+import * as S from "./ProductCommentSection.styles";
+
+const ProductCommentSection = ({ productId }) => {
+  let navigate = useNavigate();
```

Comment:
❗️ 수정요청
필요한 경우가 아니라면 const로 선언해주세요~

```suggestion
  const navigate = useNavigate();
```